book/projects/project_readmes/dzdt.md

```diff
@@ -0,0 +1,26 @@
+# dzdt
+
+## Team Members
+
+The following people contributed to our project throughout the week:
+* Project lead: [Joanmarie Del Vecchio](url to their webpage)
+* Helper: [Scott Henderson](url to their webpage)
+* Helper: [Shashank Bushan](url to their webpage)
+* Team member: [Matt Olson](url to their webpage)
+
+## Project Goals
+
+In addition to learning more about working with ICESat-2 data on the cloud, our goals are to explore
+permafrost and land surface change with crossover ATL06 data
+
+## Project Outcomes
+
+* The group has learned more about working with cloud-hosted data and vizualization technies with geodataframes
+* Joanmarie wrote a script to acquire ATL06 crossovers for dzdt 
+
```

Comment:
* Reported bugs in [sliderule example documentation](https://github.com/ICESat2-SlideRule/sliderule/issues/300) and [icepyx tutorial](https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/pull/82).